charts/proxy/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+name: athens-proxy
+version: 0.0.1
```

Comment:
Now that we have docker images published, and soon helm charts, I think we'll need to a follow-on issue/pr document for maintainers how to cut a release, as these numbers will need to be bumped as part of that.

Reply:
#567